### PR TITLE
CLN: Added missing docstring

### LINKED ISSUE #133: create_activity_flag missing correct docstring

### PR DESCRIPTION
**Pull request by henrymartin1**
fixes #133

### CODE REVIEW

File: docs/modules/preprocessing.rst [hunk 1]
```diff
@@ -30,6 +30,7 @@ transfer between two transport modes). We can cluster these into locations that
 frequently visits.
 
 .. autofunction:: trackintel.preprocessing.staypoints.generate_locations
+.. autofunction:: trackintel.preprocessing.staypoints.create_activity_flag
```
Comment (hong2223) [inline docs/modules/preprocessing.rst:33]: should we include some description of this function? for example just simply adding:

> Staypoints are points where someone stayed for a longer period of time (e.g., during a transfer between two transport modes). We can cluster these into locations that a user frequently visits **and/or infer if they correspond to activities.**

**Review by hong2223 — COMMENTED** [review]
Just an optional comment on the preprocessing documentation :). The others are fine and ready to merge

File: docs/modules/preprocessing.rst (hunk 1 above)
Comment (henrymartin1) [reply docs/modules/preprocessing.rst:33]: Yes! We should show them what we have ;-) 
Is this in the readme?

Comment (hong2223) [reply docs/modules/preprocessing.rst:33]: This is just 4 lines above :).

**Pull request merged**